impl/src/main/java/org/jboss/forge/roaster/model/impl/Strings.java

```diff
@@ -8,7 +8,7 @@
  * @author <a href="mailto:lincolnbaxter@gmail.com">Lincoln Baxter, III</a>
  *
  */
-class Strings
```

Comment:
I don't think this class is going to be accessed by any class outside of this package, WDYT?